src/components/navbar/NavbarPure.js

```diff
@@ -1,81 +1,103 @@
 import React from 'react'
 import OutsideClickHandler from 'react-outside-click-handler'
+import TranslationProvider from '../translationProvider'
+import { FormattedHTMLMessage } from 'react-intl'
+import LanguageContext from 'src/contexts/language'
+import translations from './translations'
 import logo from 'src/images/logo.svg'
 import './navbar.less'
+import configPages from 'src/config'
+import LanguageMenu from '../languageMenu'
 
 const Navbar = ({ closeMenu, toggleMenu, isMenuShown }) => {
+  const currentLanguage = LanguageContext._currentValue
+  const domainHref = 'https://platinum.fund/'
+  const hrefWithLanguage = pageName => {
+    if (
+      configPages[pageName] &&
+      configPages[pageName].languages &&
+      configPages[pageName].languages.indexOf(currentLanguage) !== -1
```

Comment:
Move it to statement please

Reply:
it makes no sense, it will add more lines of code that is not so logical

Reply:
It's better to understand thats going on

const isLanguageExist = configPages[pageName] && 
  configPages[pageName].languages && 
  configPages[pageName].languages.indexOf(currentLanguage) !== -1

if (isLanguageExist ) { ... }